specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-04-15-preview/types/experiments.json

```diff
@@ -0,0 +1,867 @@
+          "minLength": 1
+        }
+      },
+      "required": [
+        "key",
+        "value"
+      ],
+      "additionalProperties": false
+    },
+    "selector": {
+      "type": "object",
+      "description": "Model that represents a selector in the Experiment resource.",
+      "discriminator": "type",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Enum of the selector type.",
+          "enum": [
+            "List",
+            "Query"
+          ],
+          "x-ms-enum": {
+            "name": "SelectorType",
+            "modelAsString": false
+          }
+        },
+        "id": {
+          "type": "string",
+          "description": "String of the selector ID.",
+          "minLength": 1
+        },
+        "filter": {
+          "$ref": "#/definitions/filter"
+        }
+      },
+      "required": [
+        "type",
+        "id"
+      ],
+      "additionalProperties": true
+    },
+    "listSelector": {
+      "type": "object",
+      "description": "Model that represents a list selector.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/selector"
+        }
+      ],
+      "properties": {
+        "targets": {
+          "type": "array",
+          "description": "List of Target references.",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/targetReference"
+          }
+        }
+      },
+      "additionalProperties": false,
+      "x-ms-discriminator-value": "List",
+      "required": [
+        "targets"
+      ]
+    },
+    "querySelector": {
+      "type": "object",
+      "description": "Model that represents a query selector.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/selector"
+        }
+      ],
+      "properties": {
+        "queryString": {
+          "type": "string",
+          "description": "Azure Resource Graph (ARG) Query Language query for target resources."
+        },
+        "subscriptionIds": {
+          "type": "array",
+          "description": "Subscription id list to scope resource query.",
+          "items": {
+            "type": "string",
+            "description": "String of the availability zone ID."
+          },
+          "minItems": 1
+        }
+      },
+      "additionalProperties": false,
+      "x-ms-discriminator-value": "Query",
+      "required": [
+        "queryString",
+        "subscriptionIds"
+      ]
+    },
+    "filter": {
+      "type": "object",
+      "description": "Model that represents available filter types that can be applied to a targets list.",
+      "discriminator": "type",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Enum that discriminates between filter types. Currently only `Simple` type is supported.",
+          "enum": [
+            "Simple"
+          ],
+          "x-ms-enum": {
+            "name": "FilterType",
+            "modelAsString": false
```

Comment:
>false [](http://example.com/codeflow?start=29&length=5)

Same question here, why specifically change this to false?

Reply:
Changed